src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void getOperation(List<FruitTransaction> transactions) {
+        List<FruitTransaction> transactionsSelected = transactions.stream()
+                .filter(a -> a.getOperation() == FruitTransaction.Operation.RETURN)
+                .toList();
+        for (FruitTransaction fruitTransaction : transactionsSelected) {
+            int newValue =
+                    Storage.fruits.get(fruitTransaction.getFruit())

```

Comment:
There seems to be a logical error in the calculation of `newValue`. The operation is subtracting the quantity from what appears to be the current stock, which is not the intended behavior for a 'RETURN' operation. The quantity should be added to the current stock instead.